backend/src/main/java/com/greenfoxacademy/backend/services/user/UserService.java

```diff
@@ -22,4 +22,6 @@ public interface UserService extends UserDetailsService {
 
   ProfileUpdateResponseDto profileUpdate(
       String user, ProfileUpdateRequestDto profileUpdateRequestDto) throws Exception;
+
+  public void deleteProfile(String username) throws Exception;
```

Comment:
`public` should be redundant in interfaces.

Reply:
Resolved on https://github.com/gfa-cc-after/Pet-Clinic-GilmoreDevs/pull/75/commits/e0e1d3ddc3d31cfcc55070750c8f1a3580b62008